Update deploy.md with information how to deploy to render.com (#1391)
snowpackjs/astro: docs/src/pages/guides/deploy.md

Then click the **Save and Deploy** button.

## Render

> You can deploy your Astro project on [render](https://render.com/) following these steps:

Reviewer: ```suggestion
You can deploy your Astro project on [Render](https://render.com/) following these steps:
```

1. Create a [render.com account](https://dashboard.render.com/) and sign in

> 2. Click the "**new+**"-button at the dashboard and select "**Static Site**"

Reviewer: ```suggestion
2. Click the **New +** button from your dashboard and select **Static Site**
```
To match the rest of the copy on this page, we can use bold text to reference the button label; we don’t need bold _and_ quotes

3. Connect your [GitHub](https://github.com/) or [GitLab](https://about.gitlab.com/) repository or alternatively enter the public URL of a public repository

> 4. Give your website a name (1. to find it on the dashboard. 2. The onrender.com url is prefixed with it.), select the branch and specify the build command and publish directory  

Reviewer: Minor suggestion (you can ignore): I don’t think you need to give 2 reasons in parentheses why you should name the project; I think it can be inferred.

Reviewer 2: I think the point is more that you can just fully remove the blurb

Author: Do you mean removing everthing in point 4, except "build command" and "publish directory"?

Reviewer 2: oh! i was looking at old code - what's there rn looks great!

   - **build command:** `npm run build`

>    - **publish directory:** dist

Reviewer: ```suggestion
   - **publish directory:** `dist`
```

Consistency with previous line

Author: My intension was to make a difference between command and folder name.

Reviewer: That makes sense to me. But the rest of the document does style folder names as code right now. If we change that we should change all of them (outside the scope of this PR).

Author: I agree. In the context that both is filled in in a form it make sense to have both code styled.

> 6. Click the "Create Static Site" -button

Reviewer: ```suggestion
6. Click the "Create Static Site" button
```
